registry.json

```diff
@@ -10,6 +10,20 @@
  "telegram": "conraddit",
  "operatorName": "CONRAD",
  "github": "conraddit"
+},
+ {
+ "ticker": "APOLO",
+ "poolId": "c6074d596a4d1629304bfd92715ec1677aeb378e4932723616d575f1",
+ "social": {
+     "website": "https://www.apolostakepool.com",
+     "twitter": "apolostakepool",
+     "facebook": "",  
+     "instagram": "apolostakepool",
+     "youtube": ""
+ },
+ "telegram": "apolosp",
```

Comment:
Please change this to the telegram username of an individual and NOT a channel or group